js/foundation.util.mediaQuery.js

```diff
@@ -84,7 +84,7 @@ var MediaQuery = {
   _getCurrentSize: function() {
     var matched;
 
-    for (var i in this.queries) {
```

Comment:
@pelly Got it, thanks :) A little annoying to have to work around other libraries that modify built-in prototypes, but it is what it is.